web/tests/lib/namespace.js

```diff
@@ -26,7 +26,7 @@ let jquery_function;
 const template_path = path.resolve(__dirname, "../../templates");
 
 /* istanbul ignore next */
-function need_to_mock_template_error(filename) {
+function need_to_mock_template_warning(filename) {
```

Comment:
Should the function be removed entirely? I thought that the function is quite helpful in explaining how to use `mock_template` and therefore thought giving a warning in console would be better than removing the function.

Reply:
Probably a good question for the chat.zulip.org thread for more visibility, I'm not sure what the right answer is.